api/jobset/v1alpha2/jobset_types.go

```diff
@@ -84,6 +84,8 @@ type JobSetSpec struct {
 	// ReplicatedJobs is the group of jobs that will form the set.
 	// +listType=map
 	// +listMapKey=name
+	// +kubebuilder:validation:MinItems:=1
```

Comment:
Since CEL validation executes before webhook validation, do we still require `len(js.Spec.ReplicatedJobs) > 0` check for DependsOn validation ?
https://github.com/kubernetes-sigs/jobset/blob/b8b00b393938e21b4d3ac74d9748dda2d43477b3/pkg/webhooks/jobset_webhook.go#L163

Reply:
Good call.